tools/mmp/driver.cs

```diff
@@ -1371,9 +1371,9 @@ static int Compile ()
 				}
 
 				args.Append ("-liconv -x objective-c++ ");
-				if (XcodeVersion.Major > 10) {
+				if (XcodeVersion.Major >= 10) {
 					// Xcode 10 doesn't ship with libstdc++
-					args.Append ("-stdlib:libc++");
+					args.Append ("-stdlib=libc++ ");
```

Comment:
what that meant to be in the same PR ?
because the title does not imply any tooling change

Reply:
Yes, because the tests that weren't running before were failing. This fix makes those tests pass again.

I've updated the PR's description.